hutch_python/tests/test_plugins/test_questionnaire.py

```diff
@@ -42,3 +43,12 @@ def test_questionnaire_plugin():
     objs = plugin.get_objects()
     assert objs['inj_x'].run == '15'
     assert objs['inj_x'].proposal == 'LR12'
+
+
+def test_questionnaire_bad_conf():
+    logger.debug('test_questionnaire_bad_conf')
+    conf = {'experiment': {}, 'questionnaire': True}
+    plugin = Plugin(conf)
+    hutch_python.plugins.questionnaire.QSBackend = QSBackend
+    with pytest.raises(ValueError):
+        plugin.get_objects()
```

Comment:
What is this here for? Just 100% coverage?

Reply:
If it raises a different error, there is a problem...